[ENH]  adding `inv` for Laplace distribution

josephsalmon opened the pull request:
Dear all,

While creating an animation (<https://josephsalmon.github.io/HAX603X/Courses/notations.html#cas-des-variables-discr%C3%A8tes>), I realized that the `inv` (quantile) function was missing for the Laplace distribution.
The formula is given on Wikipedia <https://en.wikipedia.org/wiki/Laplace_distribution#Cumulative_distribution_function>, and is also the same function used for the sample function (since this is require for the inverse transform method) in the current code.

Hence, this PR solves this by adding
- the corresponding `inv` function itself
- the associated tests (for 4 different values of  `p,mu,b`).

I hope this helps and get merged.
Best,
Joseph

josephsalmon committed:
adding inv function to laplace distribution

josephsalmon committed:
update in inv function, renaming quantile by p and using temp x variable

kgryte commented:
If all you need is the quantile method for the Laplace distribution, you could also consider using [`@stdlib/stats-base-dists-laplace-quantile`](https://github.com/stdlib-js/stats-base-dists-laplace-quantile/tree/main), which provides that, and only that, functionality.

josephsalmon commented:
Dear @kgryte , thanks for the feedback and for making me discover https://github.com/stdlib-js/stats-base-dists
I have updated my widget accordingly and simplified my code thanks to it, and could handle more distributions thanks to it. The ouput is here, if any interest:
https://josephsalmon.github.io/HAX603X/Courses/notations.html#cas-des-variables-discr%C3%A8tes

Best,
Joseph

kgryte commented:
@josephsalmon Thanks for the link! If you have requests for other distributions, feel free to stop by the [main project](https://github.com/stdlib-js/) and we can continue the discussion there.

An eventual goal of ours is to have `jStat` eventually build on and leverage `stdlib` functionality, but we haven't had the time/resources to finish this effort. In the meantime, if there are gaps in `stdlib`, would be good to resolve first there, so that we don't duplicate effort in the future.
